### ISSUE TITLE
Guzzle 7 Uncaught Error: Call to undefined function GuzzleHttp\Psr7\stream_for()

### ISSUE THREAD

**Issue by kupoback** [kamermans/guzzle-oauth2-subscriber#31]
When I attempt to get an access token on the latest version of Guzzle, `^7.3`, it errors out giving me the following error:
```
PHP Fatal error:  Uncaught Error: Call to undefined function GuzzleHttp\Psr7\stream_for() in /path/to/project/vendor/kamermans/guzzle-oauth2-subscriber/src/GrantType/ClientCredentials.php:93
Stack trace:
#0 /path/to/project/vendor/kamermans/guzzle-oauth2-subscriber/src/GrantType/ClientCredentials.php(57): kamermans\OAuth2\GrantType\ClientCredentials->getPostBody()
#1 /path/to/project/vendor/kamermans/guzzle-oauth2-subscriber/src/OAuth2Handler.php(285): kamermans\OAuth2\GrantType\ClientCredentials->getRawData(Object(kamermans\OAuth2\Signer\ClientCredentials\BasicAuth))
#2 /path/to/project/vendor/kamermans/guzzle-oauth2-subscriber/src/OAuth2Handler.php(213): kamermans\OAuth2\OAuth2Handler->requestNewAccessToken()
#3 /path/to/project/public/FileName.php(230): kamermans\OAuth2\OAuth2Handler->getAccessTo in /path/to/project/vendor/kamermans/guzzle-oauth2-subscriber/src/GrantType/ClientCredentials.php on line 93
```

This is the method I've written, maybe I did something wrong, but dropping the Guzzle version to the latest v6, it works:

```
protected function grabToken()
{
    $client_id     = env("CLIENT_ID");
    $client_secret = env("CLIENT_SECRET");

    try {
        $reauth_client = new Client(['base_uri' => 'https://URL.com/auth/connect/token',]);

        $reauth_config = [
            'client_id'     => $client_id,
            'client_secret' => $client_secret,
        ];

        $grant_type = new ClientCredentials($reauth_client, $reauth_config);
        $refresh_gt = new RefreshToken($reauth_client, $reauth_config);
        $oauth      = new OAuth2Middleware($grant_type, $refresh_gt);

        error_log(print_r($oauth->getAccessToken(), true)); // This is me checking the token's existence

        return $oauth;
    }
    catch (\Exception $exception) {
        return rest_ensure_response($exception->getMessage());
    }
}
```

**Comment by MattApril** [comment]
Used this package many times before, but also ran into this issue today.

**Comment by stsepelin** [comment]
I fixed id by changing `stream_for` to `Utils::streamFor` in ClientCredentials.

**Comment by kamermans** [comment]
Thanks @stsepelin, do you know if `Utils::streamFor` has always been included?  Maybe `stream_for` was just an alias that was removed.

**Comment by stsepelin** [comment]
@kamermans I don't know, just saw that stream_for were removed in guzzlehttp/psr7:2.0 and replaced with Utils::streamFor.

**Comment by kamermans** [comment]
I've released the fix for this issue in [`v1.0.10`](https://github.com/kamermans/guzzle-oauth2-subscriber/releases/tag/v1.0.10).
